.github/workflows/summary.yml

```diff
@@ -0,0 +1,29 @@
+name: Notify owners with a summary about open issues and PRs
+
+on:
+  schedule:
+  - cron: "0 7 * * 1"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: create summary
+        run: |
```

Comment:
nit: Would it be nicer to add some brief comments to the run section of the (only available) step?

Reply:
Done